regression/scripts/segments/segment_test.py

```diff
@@ -427,29 +421,12 @@ def mission_setup(analyses):
     segment = Segments.Climb.Constant_CAS_Constant_Rate(base_segment)
     segment.tag = "climb_9"
     segment.analyses.extend( analyses.base )  
-    segment.altitude_end                     = 10.9   * Units.km    
+    segment.altitude_end                     = 11.   * Units.km    
     segment.calibrated_air_speed             = 150. * Units.m / Units.s
     segment.climb_rate                       = 1.  
     # add to misison
     mission.append_segment(segment)
     
-    # ------------------------------------------------------------------
```

Comment:
Come on! I just added this regression!!

Reply:
I know... I realized I missed it and that's why it failed. Adding it in